openmrs-client/src/main/java/org/openmrs/mobile/utilities/InputField.kt

```diff
@@ -0,0 +1,43 @@
+package org.openmrs.mobile.utilities
+
+import android.os.Parcel
+import android.os.Parcelable
+import java.io.Serializable
+
+
+class InputField(var concept: String) : Serializable, Parcelable {
```

Comment:
The class variable `concept` may be null as suggested at line 29
```suggestion
class InputField(var concept: String?) : Serializable, Parcelable {
```